catalog/networking/shared-vpc/sharedVPC.yaml

```diff
@@ -3,5 +3,6 @@ kind: ComputeSharedVPCHostProject
 metadata:
   namespace: networking # {"$kpt-set":"namespace"}
   annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/landing-zone:networking/v0.2.0
```

Comment:
Based on TF conventions since this is a top level pkg in catalog, shouldn't this be just `networking` as opposed to `landing-zone:networking`

Reply:
This exists as a top level package, but we expect it to be primarily used through landing zone scenarios so it's listed under there. In either case, it's still easily attributable.